### ISSUE TITLE
Segmentation fault with @extend and :global(> .sth)

### ISSUE THREAD

**Issue by tleunen** [sass/libsass#2366]
### Title: Be as meaningful as possible in 60 chars if possible

input.scss
```scss
.item {
    display: inline-block;

    :global(> .ext-link) {
        background: #000;
    }
}

.link {
    color: red;
}

.textLink {
    @extend .link;
    padding: 0 10px;
}
```

[libsass 3.5.5] [1]
```css
Segmantation fault
```

ruby sass 3.4.21
```css
Segmantation fault
```

version info:
See https://github.com/sass/node-sass/issues/1961

**Comment by mgreter** [comment]
Segfault has been fixed in libsass, but output does not match ruby sass.

**Comment by mgreter** [comment]
This is basically also a duplicate of #2055 (missing specific pseudo selector behavior)

**Comment by mgreter** [comment]
Closed via #2531
